core/chain_makers.go

```diff
@@ -276,3 +279,23 @@ func (cr *fakeChainReader) GetHeaderByHash(hash common.Hash) *types.Header
 func (cr *fakeChainReader) GetHeader(hash common.Hash, number uint64) *types.Header { return nil }
 func (cr *fakeChainReader) GetBlock(hash common.Hash, number uint64) *types.Block   { return nil }
 func (cr *fakeChainReader) StateAt(root common.Hash) (*state.StateDB, error)        { return nil, nil }
+
+type fakeEVMRunner struct {
```

Comment:
I think this is fine, but if for any reason we need some calls to succeed with certain values, we could create a fake one which you can 'program' when you create it (e.g. make it return a certain value if the method name is `getGasPriceMinimum`, or whatever.  It's not as nice as if we could mock out the higher-level function directly, but could do in a pinch.